### PR TITLE
Add AppHelper tracking screen to datalayer sample app

### PR DESCRIPTION
**Pull request by luizgrp**
#### WHAT

Add AppHelper tracking screen to datalayer sample app.

https://github.com/google/horologist/assets/878134/40e44747-a561-4be4-b724-46ba11b2ad49

#### Checklist :clipboard:
- [N/A] Add explicit visibility modifier and explicit return types for public declarations
- [x] Run spotless check
- [x] Run tests
- [N/A] Update metalava's signature text files

### CODE REVIEW

**Comment by luizgrp** [comment]
cc @ej355

**Review by yschimke — APPROVED** [review]
Quick review mainly based on the video. LGTM

File: datalayer/sample/wear/src/main/java/com/google/android/horologist/datalayer/sample/screens/tracking/TrackingScreenViewModel.kt [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.launch
+
+class TrackingScreenViewModel(
+    private val wearDataLayerAppHelper: WearDataLayerAppHelper,
+) : ViewModel() {
+
+    private val fakeTileList = listOf("Tile1", "Tile2")
+    private val fakeComplicationList = listOf("Comp1", "Comp2")
```
[the remark this answers is not in the document]
Comment (yschimke) [reply datalayer/sample/wear/src/main/java/com/google/android/horologist/datalayer/sample/screens/tracking/TrackingScreenViewModel.kt:36]: This was going to be one of my questions.

[the remark this answers is not in the document]
Comment (luizgrp) [reply datalayer/sample/wear/src/main/java/com/google/android/horologist/datalayer/sample/screens/tracking/TrackingScreenViewModel.kt:36]: I had these comments written hours ago and forgot to submit the review :D

**Pull request merged**